Queue/Queue.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+/*
+ * Algorythim analisys of Queues and it's basic operations
+ * Goal: Improve knowledge about queue data structures and improve practice
+ * 
+ * Queue implementation through Nodes
+ */
+
+namespace Queue1
+{
+    class Queue
+    {
+        public class MyQueue<TNode>
+        {
+            private Node<TNode> Front;
+            private Node<TNode> Back;
+
+            // Creates a Node with the data informed and sets the Next attr
+            private class Node<TNode>
```

Comment:
C# possui uma forma "especial" de comentários específicos para documentação de projetos, bastante útil por que o próprio visual studio usa desses comentários para gerar o inteli-sense de codigos do projeto.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments
Por exemplo:

`/// <summary>
/// method <c>draw</c> renders the point.
/// </summary>
 void draw() {...}`

Esses comentarios inicial com 3 barras